### PR TITLE
AST-1787 - New astra below, above header and mobile header showing on old header meta setting in the classic editor.

### PR DESCRIPTION
**Pull request by GauravKhupse**
### Description
If we moved to the old header with a classic editor on, it was showing a new header builder meta settings option in the old header:

### Screenshots

https://share.bsf.io/lluEdWNd

### Types of changes

Bug fix (non-breaking change which fixes an issue) 

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: inc/metabox/class-astra-meta-boxes.php [hunk 1]
```diff
@@ -265,7 +265,7 @@ public function markup_meta_box( $post ) {
 				<div class="disable-section-meta">
 					<?php do_action( 'astra_meta_box_markup_disable_sections_before', $meta ); ?>
 
-					<?php if ( $show_meta_field && Astra_Builder_Helper::is_row_empty( 'above', 'header', 'desktop' ) ) : ?>
+					<?php if ( $show_meta_field && Astra_Builder_Helper::is_row_empty( 'above', 'header', 'desktop' ) && true === Astra_Builder_Helper::$is_header_footer_builder_active ) : ?>
```
Comment (imnavanath) [inline inc/metabox/class-astra-meta-boxes.php:268]:
```suggestion
					<?php if ( $show_meta_field && true === Astra_Builder_Helper::$is_header_footer_builder_active && Astra_Builder_Helper::is_row_empty( 'above', 'header', 'desktop' ) ) : ?>
```

File: inc/metabox/class-astra-meta-boxes.php [hunk 2]
```diff
@@ -281,14 +281,17 @@ public function markup_meta_box( $post ) {
 					</div>
 					<?php endif; ?>
 
-					<?php if ( $show_meta_field && Astra_Builder_Helper::is_row_empty( 'below', 'header', 'desktop' ) ) : ?>
+					<?php if ( $show_meta_field && Astra_Builder_Helper::is_row_empty( 'below', 'header', 'desktop' ) && true === Astra_Builder_Helper::$is_header_footer_builder_active ) : ?>
```
Comment (imnavanath) [inline inc/metabox/class-astra-meta-boxes.php:284]:
```suggestion
					<?php if ( $show_meta_field && true === Astra_Builder_Helper::$is_header_footer_builder_active && Astra_Builder_Helper::is_row_empty( 'below', 'header', 'desktop' ) ) : ?>
```

File: inc/metabox/class-astra-meta-boxes.php [hunk 3]
```diff
@@ -281,14 +281,17 @@ public function markup_meta_box( $post ) {
...
 					<div class="ast-hfb-below-header-display-option-wrap">
 						<input type="checkbox" id="ast-hfb-below-header-display" name="ast-hfb-below-header-display" value="disabled" <?php checked( $below_header, 'disabled' ); ?> />
 						<label for="ast-hfb-below-header-display"><?php esc_html_e( 'Disable Below Header', 'astra' ); ?></label> <br />
 					</div>
-					<?php endif; ?>
+						<?php 
+					endif; 
+					$temp = Astra_Builder_Helper::$is_header_footer_builder_active
+					?>
```
Comment (imnavanath) [inline inc/metabox/class-astra-meta-boxes.php:292]:
```suggestion
					<?php endif; ?>
```

File: inc/metabox/class-astra-meta-boxes.php [hunk 4]
```diff
@@ -281,14 +281,17 @@ public function markup_meta_box( $post ) {
...
 
-					<?php if ( $show_meta_field && Astra_Builder_Helper::is_row_empty( 'primary', 'header', 'mobile' ) || Astra_Builder_Helper::is_row_empty( 'above', 'header', 'mobile' ) || Astra_Builder_Helper::is_row_empty( 'below', 'header', 'mobile' ) ) : ?>
+					<?php if ( $show_meta_field && true === Astra_Builder_Helper::$is_header_footer_builder_active && ( Astra_Builder_Helper::is_row_empty( 'primary', 'header', 'mobile' ) || Astra_Builder_Helper::is_row_empty( 'above', 'header', 'mobile' ) || Astra_Builder_Helper::is_row_empty( 'below', 'header', 'mobile' ) ) ) : ?>
```
Comment (imnavanath) [inline inc/metabox/class-astra-meta-boxes.php:294]: I think `Astra_Builder_Helper::is_row_empty()` condition should be in negative manner! 
If any row is not empty then we need to show "Disable" option, correct? @GauravKhupse can you please verify this case once.

File: inc/metabox/class-astra-meta-boxes.php (hunk 1 above)
Comment (GauravKhupse) [reply inc/metabox/class-astra-meta-boxes.php]: Changes done!

File: inc/metabox/class-astra-meta-boxes.php (hunk 3 above)
Comment (GauravKhupse) [reply inc/metabox/class-astra-meta-boxes.php]: Changes done!

File: inc/metabox/class-astra-meta-boxes.php (hunk 4 above)
Comment (GauravKhupse) [reply inc/metabox/class-astra-meta-boxes.php:290]: I checked, this condition is defined opposite, I will add a task in the backlog

**Review by imnavanath — APPROVED** [review]

**Comment by imnavanath** [comment]
Please add changelog for this @GauravKhupse.

**Review by nileshc-bsf — APPROVED** [review]

**Pull request merged**